data/en/lms_2.json

```diff
@@ -466,7 +466,10 @@
                             "type": "General",
                             "answers": [{
                                 "id": "dob-age-answer",
-                                "type": "Number",
+                                "unit": "duration-year",
+                                "type": "Unit",
+                                "length": "long",
+                                "use_plural_for_label": true,
```

Comment:
if default was true this would be less verbose

Reply:
Then all the other places it's used, it would be just as verbose?